src/from-scratch.js

```diff
@@ -1,16 +1,39 @@
-const resolvedWrapper = () => {
+const resolvedWrapper = (value) => {
+  return Promise.resolve(value);
 };
 
-const rejectedWrapper = () => {
+const rejectedWrapper = (errorMessage) => {
+  const error = new Error(errorMessage); // Create a new Error object with the given message
+  return Promise.reject(error);
 };
 
-const handleResolvedPromise = () => {
+const handleResolvedPromise = (value) => {
+  return value.then((value) => {
+    console.log(value);
+    return value.toUpperCase();
+  });
 };
 
-const handleResolvedOrRejectedPromise = () => {
+// Handles both resolved and rejected Promises, ensuring consistent output in either case.
+const handleResolvedOrRejectedPromise = (value) => {
+  return value
+    .then((value) => {
+      console.log(value);
+      return value.toUpperCase();
+    })
+    .catch((err) => {
+      console.error(`Your error message was: ${err.message}`);
+      return null;
+    });
 };
 
-const pauseForMs = () => {
+// Best practice: Use `pauseForMs` in async/await workflows for better readability.
```

Comment:
Can you clarify what you mean by this? I'm not sure I understand.

Also, for what its worth, there doesn't need to be a best practice shared in this assignment, in case you might have been searching for one to share.